docs/index.rst

```diff
@@ -7,6 +7,39 @@
 Welcome to bodhi's documentation!
 =================================
 
+What is Bodhi?
+--------------
+Bodhi is a modular web-based system that facilitates the process of publishing package updates for Fedora. Bodhi is the application that manages updates for Fedora releases. It maintains a single stage of repositories by adding/updating/removing packages. It can be used to create or modify updates and overrides.
+
+Advantages of bodhi
+-------------------
```

Comment:
I would probably rename this section "Features". Also, I recommend using bullet points instead of numbers below.